<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-I4RR6GHR" failures="1" tests="12" name="Comments Tests" time="62.14" errors="0" timestamp="2021-08-19T19:14:38 CST">
  <testcase classname="CommentsTests" name="Create_Comment_Successful" time="18.31"/>
  <testcase classname="CommentsTests" name="Create_Comment_UnSuccessful" time="4.688"/>
  <testcase classname="CommentsTests" name="Delete_Comment_Successful" time="6.149"/>
  <testcase classname="CommentsTests" name="Delete_Comment_UnSuccessful" time="0.95"/>
  <testcase classname="CommentsTests" name="Test_GET_Comment" time="12.604"/>
  <testcase classname="CommentsTests" name="Test_GET_Comments" time="2.65"/>
  <testcase classname="CommentsTests" name="Test_GET_Comments_Invalid" time="2.783"/>
  <testcase classname="CommentsTests" name="Test_GET_Invalid_Comment" time="0.778"/>
  <testcase classname="CommentsTests" name="Update_Comment" time="5.027"/>
  <testcase classname="CommentsTests" name="Update_Invalid_Comment" time="0.758">
    <failure type="java.lang.AssertionError" message="1 expectation failed.
Expected status code &amp;lt;406&amp;gt; but was &amp;lt;200&amp;gt;.
">
      <![CDATA[java.lang.AssertionError: 1 expectation failed.
Expected status code <406> but was <200>.

at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrap.callConstructor(ConstructorSite.java:84)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:277)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:493)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$2.call(Unknown Source)
at io.restassured.internal.ResponseSpecificationImpl.validateResponseIfRequired(ResponseSpecificationImpl.groovy:674)
at org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)
at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:193)
at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:61)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)
at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:126)
at io.restassured.specification.ResponseSpecification$statusCode$0.callCurrent(Unknown Source)
at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:134)
at io.restassured.internal.ValidatableResponseOptionsImpl.statusCode(ValidatableResponseOptionsImpl.java:89)
at io.restassured.internal.ValidatableResponseImpl.super$2$statusCode(ValidatableResponseImpl.groovy)
at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)
at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1268)
at org.codehaus.groovy.runtime.ScriptBytecodeAdapter.invokeMethodOnSuperN(ScriptBytecodeAdapter.java:144)
at io.restassured.internal.ValidatableResponseImpl.statusCode(ValidatableResponseImpl.groovy:142)
at io.restassured.internal.ValidatableResponseImpl.statusCode(ValidatableResponseImpl.groovy)
at CommentsTests.Update_Invalid_Comment(CommentsTests.java:173)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- Update_Invalid_Comment -->
  <testcase classname="CommentsTests" name="createComment" time="2.254"/>
  <testcase classname="CommentsTests" name="createPost" time="1.568"/>
</testsuite> <!-- Comments Tests -->
